MatrixSDK/JSONModels/Aggregations/MXAggregationPaginatedResponse.m

```diff
@@ -35,12 +35,18 @@ + (instancetype)modelFromJSON:(NSDictionary *)JSONDictionary
     MXAggregationPaginatedResponse *paginatedResponse;
 
     NSArray<MXEvent*> *chunk;
-    MXJSONModelSetMXJSONModelArray(chunk, MXEvent.class, JSONDictionary[@"chunk"])
-
+    NSArray *chunkJson = JSONDictionary[@"chunk"];
+    MXJSONModelSetMXJSONModelArray(chunk, MXEvent.class, chunkJson)
+    // For some reason modelsFromJSON returns nil if you pass it an empty array.
```

Comment:
Really a more sensible behaviour for `modelsFromJSON` would be to return an empty array if you pass it an empty array. But probably too risky a change, considering this repo is mostly deprecated, so just special-cased it here.